packages/sitecore-jss/src/editing/utils.ts

```diff
@@ -74,8 +74,12 @@ export class HorizonEditor {
     if (isServer()) {
       return false;
     }
-    // Horizon will add "sc_horizon=editor" query string parameter for the editor and "sc_horizon=simulator" for the preview
-    return window.location.search.indexOf('sc_horizon=editor') > -1;
+    // XMCloud Pages (ex-Horizon) will have hrz-canvas-state element present in Edit and Preview
+    // But Edit would not have the have the sc_horizon query string param
+    return (
```

Comment:
We still need to support old "Chromes" method and the check
`window.location.search.indexOf('sc_horizon=editor') > -1` still should be here. Additionally, Stas and I discussed the idea that we can add our own script and define a global variable there (as an example) so we will not rely on Pages API.
"hrz-canvas-state" is rendered by us in [EditingScripts](https://github.com/Sitecore/jss/blob/dev/packages/sitecore-jss-react/src/components/EditingScripts.tsx#L19), let's add a new script there

Reply:
The name of the global variable can be defined in "utils" as well and used by "EditingScripts"